src/api/special-attributes.md

```diff
@@ -56,15 +56,27 @@
 
 - **Expects:** `string | Object (component’s options object)`
 
-Used for [dynamic components](../guide/component-dynamic-async.html).
+  Used for [dynamic components](../guide/component-dynamic-async.html).
 
-For example:
+  For example:
+
+  ```html
+  <!-- component changes when currentView changes -->
+  <component :is="currentView"></component>
+  ```
+
+- **Usage on native elements** <Badge text="3.1+" />
 
-```html
-<!-- component changes when currentView changes -->
-<component :is="currentView"></component>
-```
+  When the `is` attribute is used on native HTML elements, it will be interpreted as [Customized built-in elements](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-customized-builtin-example) which is a native web platform feature.
```

Comment:
```suggestion
  When the `is` attribute is used on a native HTML element, it will be interpreted as a [Customized built-in element](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-customized-builtin-example), which is a native web platform feature.
```